pkg/kubecost/allocation.go

```diff
@@ -65,16 +65,15 @@ type Allocation struct {
 	GPUCostAdjustment      float64               `json:"gpuCostAdjustment"`
 	NetworkCost            float64               `json:"networkCost"`
 	LoadBalancerCost       float64               `json:"loadBalancerCost"`
-	PVByteHours            float64               `json:"pvByteHours"`
-	PVCost                 float64               `json:"pvCost"`
+	PVBreakdown            map[string]PVUsage    `json:"pvBreakDown"`
```

Comment:
What do you think about just calling this `PVs`? Not sure what "Breakdown" adds, and I could see us adding information to this, turning it into a more robust record of the related PVs.

Reply:
Also, do you have strong feelings that this should be a value type `PVUsage` as opposed to a pointer-type `*PVUsage`? Might make things _slightly_ more performant to use `map[string]*PVUsage`.

Reply:
Also also, I think we might need to have this key off `Cluster` and `Name`, so probably going to need something like
```go
map[pvKey]PVUsage

type pvKey struct {
    cluster string
    name    string
}
```